docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -0,0 +1,206 @@
+
+
+### RBAC
+If your cluster has RBAC enabled add `ClusterRole` and `ClusterRoleBindings`. For `ClusterRoleBindings` you should update `REPLACE_WITH_NAMESPACE` value to match with namespace where the Sematext agent will be installed:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sematext-agent
+  labels:
+    app: sematext-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: sematext-agent
+subjects:
+- kind: ServiceAccount
+  name: sematext-agent
+  namespace: <REPLACE_WITH_NAMESPACE>
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sematext-agent
+  labels:
+    app: sematext-agent
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - pods
+  - configmaps
+  - nodes
+  - secrets
+  verbs:
+  - list
+  - get
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - configmaps
+  verbs:
+  - create
+  - delete
+  - update
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - watch
+  - list
+```
+### Create and deploy the DaemonSet
+
+Create a file for the deployment `st-agent-ds.yml` (Replace Tokens and receiver URLs):
```

Comment:
Create a file for the deployment `st-agent-ds.yml` => Create a `st-agent-ds.yml` file for the deployment